console/src/main/webapp/manager/app/components/user/user.es6

```diff
@@ -72,6 +72,7 @@ class UserController {
     // Load role infos for every tab (for confirmation)
     let Role = this.$injector.get('Role')
     this.roles = Role.query()
+    this.systemRoles = Role.query()
```

Comment:
There's already a duplicate call to `Role.query()`, I don't think it's a good idea to add one more…

Reply:
From template, user.roles doest not contain description, only CN. You suggest to clone instead of call Role.query twice ?

Reply:
There should be a single call (current code shouldn't have 2) to `Role.query` and get what you want in a callback, see https://github.com/georchestra/georchestra/blob/19b1c9fc0b688b1e6bf7791dde2162f902db1fe6/console/src/main/webapp/manager/app/components/user/user.es6#L76